docs/testplans/ras/ras-ingestion-test-plan.md

```diff
@@ -0,0 +1,41 @@
+# Test plan for Gen3's implementation of Researcher Auth Services (RAS)
+
+## One-liner overview of the service
+The Resercher Auth Services (RAS) is an implementation of the Global Alliance for Genomics & Health (GA4GH) Passport Specifications ([https://github.com/ga4gh-duri/ga4gh-duri.github.io/blob/master/researcher_ids/ga4gh_passport_v1.md](https://github.com/ga4gh-duri/ga4gh-duri.github.io/blob/master/researcher_ids/ga4gh_passport_v1.md)). It defines a common process for Authentication, Authorization and Logging around NIH's open and controlled data assets and repositories. This test plan focuses on the Gen3 implementation of such spec.
+More info:  ([https://auth.nih.gov/docs/RAS/serviceofferings.html](https://auth.nih.gov/docs/RAS/serviceofferings.html))
+
+## Environmental dependencies
+In order to execute this test, the environment must have the following items fully configured:
+- Latest fence version with the RAS HTTP API endpoints
+- A number of test users with their corresponding permissions (as per the table found in the next section). These test accounts must be long-living accounts in the dev tier so we can use for continuous integration and ongoing testing.
+
+## Test Plan
+
+The initial coverage comprises the following scenarios described in the testing matrix of users, access and assertions:
+
+:information_source: Due to the hard dependency on external entities (e.g., Fence will communicate with a Passport Clearing House to validate authZ/authN visas), in this first iteration, the tests will follow a semi-automated approach through executable tests (later we should mock the 3rd party services and work with dummy data).
```

Comment:
Fence will communicate with RAS (a passport / visa assertion repository) to validate authN/authZ visas